common/src/main/java/org/spine3/samples/lobby/common/util/testdata/TestDataFactory.java

```diff
@@ -60,64 +62,74 @@ private TestDataFactory() {
      * Creates a new {@link BoundedContext} instance with {@link InMemoryStorageFactory},
      * {@link CommandBus} and {@link EventBus}.
      */
+    @VisibleForTesting
     public static BoundedContext newBoundedContext() {
         final InMemoryStorageFactory storageFactory = InMemoryStorageFactory.getInstance();
         final EventStore eventStore = EventStore.newBuilder()
-                .setStreamExecutor(MoreExecutors.directExecutor())
-                .setStorage(storageFactory.createEventStorage())
-                .build();
+                                                .setStreamExecutor(MoreExecutors.directExecutor())
+                                                .setStorage(storageFactory.createEventStorage())
+                                                .build();
         final CommandBus commandBus = newCommandBus(storageFactory);
+        final String name = "BC-" + newUuid();
         final BoundedContext.Builder result = BoundedContext.newBuilder()
-                .setName("Orders & Registrations tests")
-                .setStorageFactory(storageFactory)
-                .setCommandBus(commandBus)
-                .setEventBus(EventBus.newInstance(eventStore));
+                                                            .setName(name)
+                                                            .setStorageFactory(storageFactory)
+                                                            .setCommandBus(commandBus)
+                                                            .setEventBus(EventBus.newBuilder()
+                                                                                 .setEventStore(eventStore)
+                                                                                 .build());
         return result.build();
     }
 
     /**
      * Creates a new command bus with {@link InMemoryStorageFactory}.
      */
+    @VisibleForTesting
     public static CommandBus newCommandBus() {
         final InMemoryStorageFactory storageFactory = InMemoryStorageFactory.getInstance();
         return newCommandBus(storageFactory);
     }
 
     private static CommandBus newCommandBus(StorageFactory storageFactory) {
         final CommandStore store = new CommandStore(storageFactory.createCommandStorage());
-        final CommandBus.Builder builder = CommandBus.newBuilder();
-        builder.setCommandStore(store);
-        // TODO:2016-04-08:alexander.litus: change to App Engine-compatible scheduler
-        builder.setScheduler(new ExecutorCommandScheduler());
-        return builder.build();
+        return CommandBus.newInstance(store);
```

Comment:
Check if we don't loose anything deleting `builder.setScheduler(new ExecutorCommandScheduler());`